### ISSUE TITLE
Configuration file is not serializable 

### ISSUE THREAD

**Issue by JJCLane** [optimistdigital/nova-blog#17]
After installing the package, we get `Your configuration files are not serializable.`

This is because of the `page_url` function:
```php
'page_url' => function (OptimistDigital\NovaBlog\Models\Post $post) {
        return env('APP_URL').'/blog/'.$post->slug;
},
```

**Comment by JJCLane** [comment]
For anyone looking, I solved this for now by using a string reference to a function:

```php
'page_url' => 'Site\Blog\Blog::getBlogUrl',


<?php

namespace Site\Blog;

use OptimistDigital\NovaBlog\Models\Post;

class Blog
{
    public static function getBlogUrl(Post $post)
    {
        return env('APP_URL').'/blog/'.$post->slug;
    }
}

```

**Comment by KasparRosin** [comment]
Hey!
Thank you for reporting this issue.
I've updated the config file and README and released [version 9.2.1](https://github.com/optimistdigital/nova-blog/compare/9.2.0...9.2.1) that includes the fix.
